### PR TITLE
Computing and visualizing gene evidence overlap

### PR DESCRIPTION
**Pull request by gwaygenomics**
Pull request adds script that generates Venn diagram of candidate genes based on genomic "window". Will show how many genes are selected based on LD, nearest gene, and TAD based aggregation methods.

Text files are added to the `gestalt` folder

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .Rhistory
 data/
 tad_pathways_data.tar.gz
+figures/
```
Comment (dhimmel) [inline .gitignore]: Why not track figures? Is this repo inputs only?

File: scripts/gene_evidence_venn.R [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+bmd_nearest_genes <- c()
+for (gene in bmd_genes) {
+  gene <- unlist(strsplit(gene, ","))
+  bmd_nearest_genes <- c(bmd_nearest_genes, gene)
+}
+bmd_nearest_genes <- unique(bmd_nearest_genes)
+
+# TAD based genelist
+tad_genes_df <- readr::read_tsv(tad_genes_file)
+tad_genes_df <- tad_genes_df[tad_genes_df$gene_type == "protein_coding", ]
```
Comment (dhimmel) [inline scripts/gene_evidence_venn.R]: In future, consider dplyr / piping for dataframe manipulations.

File: .gitignore (hunk 1 above)
Comment (gwaygenomics) [inline .gitignore]: no, I didn't think it necessary to add the resulting figure from this change into the repo. I could see a figure in the future though, so I will remove the line from the ignore file

**Comment by gwaygenomics** [comment]
thanks for comments @dhimmel - updated to dplyr filtering and added `.pdf` files for the two generated figures

**Pull request merged**